### PR TITLE
Madhu mng

### PR DESCRIPTION
**Pull request by MadhuMNG**
## The Code

The correct link of the image has to be considered
- [ ] Fenny
- [ ] Haneefa
- [ ] Marie
- [ ] Nazgul

### CODE REVIEW

**Comment by MadhuMNG** [comment]
npm run error

File: peoplenew/alexanderism55.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# This is a little introduction of myself
```
Comment (bermarte) [inline peoplenew/alexanderism55.md:1]: This file was ok, you don't need to add it

File: peoplenew/dnyandeo33.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# About myself
```
Comment (bermarte) [inline peoplenew/dnyandeo33.md:1]: You don't need to create a new directory, `people` was ok.

File: peoplenew/fennybio.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# _Meet Fenny_
+
+An island monkey who turns to civilization to learn how to `code` and
+`adult-ing`. I've spent half of my life jumping from one island to the other
+island then make a bigger leap from one continent to the other (hence island
+monkey) :laughing:
+
+![chimpsIMG](https://github.com/lab-brussels-1/Group-1-Project-Management/blob/master/photos/Island%20peeps.png?raw=true)
```
Comment (bermarte) [inline peoplenew/fennybio.md:8]: Use relative paths

File: peoplenew/mariabio.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# I love modern `ART`. I am for `PEACE`. I learn `CODING`
+
+![alt text](https://cdn.shopify.com/s/files/1/0969/9128/products/haring_banner_52b3eb25-7354-4ef1-ab14-ecf3b8f915b7_large.jpg?v=1507970285)
```
Comment (bermarte) [inline peoplenew/mariabio.md:3]: Did you change the image?

File: peoplenew/nazgulbio.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
+# Hi dears
+
+![space](https://e7.pngegg.com/pngimages/491/228/png-clipart-astronaut-outer-space-party-astronaut-food-label-thumbnail.png)
```
Comment (bermarte) [inline peoplenew/nazgulbio.md:3]: Idem

**Review by bermarte — COMMENTED** [review]
I have added some comments, it is better to close this PR without merging.

**Comment by bermarte** [comment]
Close the PR, check out the video.

**Pull request closed without merging**